app/controllers/content_types/modular_controller.rb

```diff
@@ -247,7 +247,7 @@ def two_column
         "SOCIAL ACCOUNTS" => "TODO: _cascade/formats/level/social_accounts",
 
         # TODO: convert these to cascade_format action.
```

Comment:
The Two Column Template in the Modular Controller was still using an older Omninav design from a previous dev team. I had to add the current production v2 design as a partial in order to test offsetting its height when toggling the accordions and scrolling to the respective accordion section

Reply:
ok makes sense